collect_app/src/main/java/org/odk/collect/android/activities/DataManagerList.java

```diff
@@ -195,7 +195,7 @@ private void createDeleteInstancesDialog() {
         mAlertDialog = new AlertDialog.Builder(this).create();
         mAlertDialog.setTitle(getString(R.string.delete_file));
         mAlertDialog.setMessage(getString(R.string.delete_confirm,
-                mSelected.size()));
+                String.valueOf(mSelected.size())));
```

Comment:
Just so I can understand the issue... Reading the docs, %s will print 'null' if null and obj.toString() otherwise. Why is this necessary?